ksql-engine/src/main/java/io/confluent/ksql/KsqlEngine.java

```diff
@@ -617,6 +615,8 @@ public QueryIdGenerator getQueryIdGenerator() {
         Collections.emptyMap(),
         metaStoreCopy
     );
+    engineMetrics.registerQueries(queryMetadataList);
```

Comment:
Can `engineMetrics.registerQueries(queryMetadataList);` be moved into `planQueries` so its in a single place?

Reply:
This will be updated after merging #1599.